packages/teleport/src/Discover/Shared/Timeout.tsx

```diff
@@ -1,37 +1,37 @@
 import React, { useEffect, useState } from 'react';
-import styled from 'styled-components';
 
 interface TimeoutProps {
   timeout: number; // ms
+  message?: string;
+  // mt is to specify margin top in pixels.
+  mt?: number;
```

Comment:
This confused me as you had it set to `100` and I thought that was the regular `mt={100}`, which would've been huge. Maybe we should wrap the usage of `<Timeout />` in a `<Box mt={4}>` or something?

Reply:
We could probably change `<Timeout />` to return a `<span>` instead then too. It's only really meant to give the text, so having the usage wrapped in a box would be a good approach.